src/models/User.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace App\Models;
+
+use App\Exceptions\UserNotSavedException;
+use PDO;
+
+class User
+{
+    protected $user;
+    protected $db;
+
+    public function __construct($config)
+    {
+        $dbconfig = $config['database'];
+        $dsn      = 'mysql:host=' . $dbconfig['host'] . ';dbname=' . $dbconfig['name'];
+        $this->db = new PDO($dsn, $dbconfig['user'], $dbconfig['pass']);
+        $this->db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    }
+
+    public function showMe()
```

Comment:
This accepts no argument, but it should. Using the session here wouldn't be preferable, because the domain should have no knowledge of the fact that this is an HTTP app.